tools/scripts/build.sh

```diff
@@ -15,7 +15,7 @@ cargo --version
 
 case $TARGET in
```

Comment:
I don't think this file is being used anymore, but that doesn't hold up the merge.

Reply:
I've given it the axe.  It's a dead parrot.